kratos/utilities/entities_utilities.cpp

```diff
@@ -17,11 +17,144 @@
 
 // Project includes
 #include "utilities/entities_utilities.h"
+#include "utilities/string_utilities.h"
+#include "utilities/geometry_utilities.h"
 
-namespace Kratos
+namespace Kratos::EntitiesUtilities
 {
-namespace EntitiesUtilities
+
+template<class TEntity>
+EntitityIdentifier<TEntity>::EntitityIdentifier(const std::string& rName)
+    : mName(rName)
```

Comment:
i do not fully understand this code, so i would appreciate if you explain a bit in words what you are trying to accomplish.

However i am under the impression that you are assuming that registered elements follow a strict naming convention.
what happens if you have a nonstandard name? say that i have something registered as "MyShell" what would happen in that case?

Reply:
I will extend description, but if the element/condition is not registered it throws an error

Reply:
Definition extended